src/main/java/edu/ucsb/ucsbcslas/controllers/CourseController.java

```diff
@@ -118,6 +118,29 @@
     List <Course> courseList = courseRepository.findAll();
     ObjectMapper mapper = new ObjectMapper();
 
+    for(Course i : courseList){
+      String activeQuarter = i.getQuarter();
+      if(activeQuarter.length() == 5) {
+        String quarter = activeQuarter.substring(4);
+        String year = activeQuarter.substring(2,4);
+        if(quarter.equals("1")){
+          quarter = "W";
+        }
+        else if(quarter.equals("2")){
+          quarter = "S";
+        }
+        else if(quarter.equals("3")){
+          quarter = "M";
+        }
+        else {
+          quarter = "F";
+        }
+        i.setQuarter(quarter+year);
+      } else {
+        i.setQuarter(Character.toUpperCase(activeQuarter.charAt(0))+activeQuarter.substring(1));
+      }
```

Comment:
Consider doing the earlier refactor, and then replacing lines 122-141 with a single line of code:

```
i.setQuarter(ActiveQuarterController.fixQuarter(i.getQuarter));
```